src/styles/blocks/compare-bike.scss

```diff
@@ -0,0 +1,63 @@
+@import "../mixins.scss";
+
+.compare {
+
+  &__item {
+    margin-bottom: 40px;
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+
+}
+
+.bike {
+
+  &__photo-container {
+    max-width: 392px;
+    margin: 0 auto 32px;
+
+    @include onTablet {
+      display: grid;
+      grid-template-columns: repeat(4, 1fr);
+      column-gap: 24px;
+      margin-bottom: 32px;
+    }
+  }
+
+  &__photo {
+    width: 100%;
+    grid-column: 1 / 5;
+
+    &:hover {
+      transition: all 0.3s ease-in-out;
```

Comment:
If you want the animation to be with a transition in both directions, you need to add it to the element, not on the hover of this element
Now if you remove the cursor from the image, it instantly returns to its original position without any transition